acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/utils/MetadataFileHelpers.java

```diff
@@ -179,6 +184,49 @@ public void createAzureKeyYamlFileAt(
     }
   }
 
+  public void createYubiHsmYamlFileAt(
+      final Path metadataFilePath,
+      final Path destinationYubiShellSimulator,
+      final KeyType keyType) {
+    final int opaqueObjId = keyType == KeyType.BLS ? 1 : 2;
+
+    final Map<String, Serializable> yaml =
+        Map.of(
```

Comment:
Map.of is quick, but hard to read - can we change it to be 2 per line, or just use the map.put?

Reply:
agreed.